test/testcafe/spec/GenericErrorHandling_spec.js

```diff
@@ -0,0 +1,43 @@
+import { RequestMock } from 'testcafe';
+import TerminalPageObject from '../framework/page-objects/TerminalPageObject';
+import xhr403SecurityAccessDenied from '../../../playground/mocks/data/idp/idx/error-403-security-access-denied';
+
+const securityAccessDeniedMock = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(xhr403SecurityAccessDenied, 403);
+
+const noMessageResponse = JSON.parse(JSON.stringify(xhr403SecurityAccessDenied));
+delete noMessageResponse.messages;
```

Comment:
not fan of manipulate data. for this case, you probably can just create brand new message.